docs/workflows/input_output.md

```diff
@@ -87,17 +51,16 @@ Output from the task application code is returned in and placed in the `payload`
 
 As of Cumulus > 1.15 and CMA > v1.1.1, configuration of the CMA is expected to be driven by AWS Step Function Parameters.
 
-Using the CMA package with the lambda by by any of the above mentioned methods (Manual, Kes, Lambda Layers) requires configuration for it's various features via a specific Step Function Parameters configuration format:
+Using the CMA package with the lambda by by any of the above mentioned methods (Manual, Kes, Lambda Layers) requires configuration for it's various features via a specific Step Function Parameters configuration format (see sample workflows in the [examples cumulus-tf source](https://github.com/nasa/cumulus/tree/master/example/cumulus-tf) for more examples):
 
-```yaml
-StepFunction:
-  Parameters:
-    cma:
-      event.$: '$'
-      configuration keys: ...
+```json
```

Comment:
Can we update this to be a full JSON object to satisfy the Github markdown parser?

Reply:
88034bcbb